platforms/ios/HelloCordova/Plugins/WizCanvasPlugin/WizCanvasView.m

```diff
@@ -235,13 +235,18 @@ - (BOOL)shouldAutorotateToInterfaceOrientation:(UIInterfaceOrientation)orientati
 #pragma mark Script loading and execution
 
 - (BOOL)loadRequest:(NSString *)url {
-    NSLog(@"Loading Script: %@", url );
+    NSLog(@"Loading Script: %@", url);
 
-    NSError *error;
+    NSError *error = nil;
     NSData *urlData = [NSData dataWithContentsOfURL:[NSURL URLWithString:url] options:0 error:&error];
 
-    if (!urlData || error) {
-        NSLog(@"Error: Can't Find Script %@", url );
+    if (error) {
+        NSLog(@"Error: Could not load the requested url %@: %@", url, [error localizedDescription]);
```

Comment:
I tested it and it works. I could not find any reason why it would be deprecated. I'll merge this soon if there is no feedback.